src/constants.ts

```diff
@@ -1,34 +1,28 @@
 import { TokenBase } from './types'
 import { Big as BigDecimal } from 'big.js'
-import { AssetId } from './types/common'
-import { zenlinkAssetIdToCurrencyId } from './utils/token'
 
-// Probably need our own tokens
-export const TOKEN_METADATA_MAP: { [address: string]: TokenBase } = {
-    '2124-0-0': { name: 'AMPLITUDE', symbol: 'AMPE', decimals: 12 },
-    '2124-2-256': { name: 'Kusama', symbol: 'KSM', decimals: 12 },
-    '2124-2-512': { name: 'Stellar Native', symbol: 'XLM', decimals: 12 },
-    '2124-2-513': { name: 'Stellar USDC', symbol: 'USDC', decimals: 12 },
-    '2124-2-514': { name: 'Stellar TZS', symbol: 'TZS', decimals: 12 },
-    '2124-2-515': { name: 'Stellar BRL', symbol: 'BRL', decimals: 12 },
```

Comment:
I removed them because only AMPE and KSM are really needed. However, you have a point in that we can also expose the data for the rest, it would not do any harm. I will re-add them.

Reply:
I re-added the other currencies now. Please have another look @adelarja